amlip_cpp/include/amlip_cpp/types/GenericDataType.hpp

```diff
@@ -232,6 +232,11 @@ class GenericDataType : public InterfaceDataType
     static const size_t DEFAULT_PREALLOCATED_SIZE_;
 };
 
+//! \c GenericDataType to stream serializator
+std::ostream& operator <<(
+        std::ostream& os,
+        const GenericDataType& obj);
```

Comment:
Add `%ignore eprosima::amlip::types::operator <<(std::ostream &,const GenericDataType&);` to `GenericDataType.i`.